lib/ansible/galaxy/collection/__init__.py

```diff
@@ -1124,6 +1127,9 @@ def _build_files_manifest_distlib(b_collection_path, namespace, name, manifest_c
             'recursive-include plugins */**.py */**.license',
         ])
 
+        if license_file:
```

Comment:
Should this be 
```suggestion
        if license_file is not None:
```
? What happens if this is an empty string? Should it be validated somewhere?

Reply:
If it's an empty string, we still don't want to add the manifest directive. That would result in an empty include directive.